video/out/gpu/d3d11_helpers.c

```diff
@@ -249,8 +254,7 @@ bool mp_d3d11_list_or_verify_adapters(struct mp_log *log,
     }
 
     if ((picked_adapter = get_d3d11_adapter(log,
-                                            adapter_name ?
-                                            (char *)adapter_name->start : NULL,
+                                            adapter_name,
                                             listing)))
     {
```

Comment:
Style nit: The if condition and following brace can fit on one line now.

Reply:
Good catch.